<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeniorAdminWindow</class>
 <widget class="QMainWindow" name="SeniorAdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>979</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="users_table_tab">
       <attribute name="title">
        <string>View users table</string>
       </attribute>
       <widget class="QPushButton" name="get_users_table_button">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>211</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Get users table</string>
        </property>
       </widget>
       <widget class="QScrollArea" name="users_table_scroll">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>951</width>
          <height>491</height>
         </rect>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>949</width>
           <height>489</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="hospitals_table_tab">
       <attribute name="title">
        <string>View hospitals table</string>
       </attribute>
       <widget class="QScrollArea" name="hospitals_table_scroll">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>951</width>
          <height>491</height>
         </rect>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>949</width>
           <height>489</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="get_hospitals_table_button">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>211</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Get hospitals table</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="junior_administrator_tab">
       <attribute name="title">
        <string>Add junior administrator</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Data enter</string>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="last_name_label">
             <property name="text">
              <string>Last name</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="last_name_form"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="first_name_text">
             <property name="text">
              <string>First name</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="first_name_form"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="middle_name_text">
             <property name="text">
              <string>Middle name</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="middle_name_form"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="phone_number_text">
             <property name="text">
              <string>Phone number</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="phone_number_form"/>
           </item>
           <item row="4" column="0" colspan="2">
            <widget class="QLabel" name="error_add_junior">
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 0, 0);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="add_junior_administrator_button">
          <property name="text">
           <string>Add junior administrator</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="new_hospital_tab">
       <attribute name="title">
        <string>Add new hospital</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Data enter</string>
          </property>
          <layout class="QFormLayout" name="formLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="region_text">
             <property name="text">
              <string>Region</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="region_form"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="settlement_type_text">
             <property name="text">
              <string>Settlement type</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="settlement_type_form"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="settlement_name_text">
             <property name="text">
              <string>Settlement name</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="settlement_name_form"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="street_text">
             <property name="text">
              <string>Street</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="street_form"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="house_text">
             <property name="text">
              <string>House</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="house_form"/>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="hospital_full_name_text">
             <property name="text">
              <string>Hospital full name</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLineEdit" name="hospital_full_name_form"/>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="id_of_junior_administrator_text">
             <property name="text">
              <string>Id of junior administrator</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLineEdit" name="id_of_junior_administrator_form"/>
           </item>
           <item row="7" column="0" colspan="2">
            <widget class="QLabel" name="error_add_hospital">
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 0, 0);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="add_new_hospital_button">
          <property name="text">
           <string>Add new hospital</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
